### ISSUE TITLE
About FPGA

### ISSUE THREAD

**Issue by sinaasadiyan** [anerisheth19/CNN-MobileNet-V1-implementation-on-AWS-FPGA-using-OpenCL#1]
Hello,
Thanks for sharing this valuable implementation.
Is the FPGA kernel.cl file the same as in the respository?
Are fpga optimizations applied to that?
and may you please explain about quantization process, are they 8 bit intiger quantized with tf lite?

Thanks

**Comment by Ushma30** [comment]
Hey!
The kernel.cl file is the same that is present in the repository but this was a very naive implementation. I have thereafter optimized it further and implemented the float version and 8 - bit quantized version as well. The code here doesn't handle the data type overflow neither we have applied any optimizations to it since we were trying to achieve a baseline implementation. 

Quantized MobileNet implemented in GPU
https://github.com/Ushma30/MobileNet-V1/tree/MobileNet-V1-Quantized
The above network that we implemented uses gemmlowp technique to quantize the resultant feature map after every layer. We are also testing this implementation with FPGA and trying to add a few optimizations to it.
